src/Orleans/LogConsistency/ILogConsistencyProvider.cs

```diff
@@ -30,14 +30,20 @@ public interface ILogViewAdaptorFactory
         /// <summary> Returns true if a storage provider is required for constructing adaptors. </summary>
         bool UsesStorageProvider { get; }
 
+
+        /// <summary> Returns true if a event-storage provider is required for constructing adaptors. </summary>
+        bool UsesEventStorageProvider { get; }
```

Comment:
Oh definitely. I'm wondering if we should hold off until we can integrate that work. We expect that to come in the 1.6 timeframe. I recognize that serializing behind future changes is often a trap - but I believe we are quite sure that feature is coming.